<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CubeIn</class>
 <widget class="QDialog" name="CubeIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CubeIn</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>661</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>611</width>
      <height>621</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/files/indoor.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="serverBtn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pathBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>91</width>
      <height>72</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>51</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>82</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>91</width>
      <height>72</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>339</y>
      <width>51</width>
      <height>133</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>51</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>51</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>530</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>51</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>51</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>120</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mekanBtn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mekan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z06">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>91</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z11">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>22</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lab1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>21</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kantin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z23">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>490</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>19</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lab2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>18</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lab3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>17</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z02">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>13</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z05">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>47</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z10">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>16</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z04">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>14</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lab4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>50</y>
      <width>30</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
